engine.py

```diff
@@ -124,26 +135,31 @@ def post_app_init(self):
             # Setup the OTLs that need to be loaded for the Toolkit apps
             self._load_otls(oplibrary_path)
 
-        # startup Qt
-        from tank.platform.qt import QtGui
-        from tank.platform.qt import QtCore
+        # no integrated pyside support. need to run custom event loop. see
+        # python/tk_houdini/python_qt_houdini.py
+        if not self._integrated_pyside:
 
-        app = QtGui.QApplication.instance()
-        if app is None:
-            # create the QApplication
-            sys.argv[0] = "Shotgun"
-            app = QtGui.QApplication(sys.argv)
-            app.setQuitOnLastWindowClosed(False)
-            app.setApplicationName(sys.argv[0])
+            # startup Qt
+            from tank.platform.qt import QtGui
 
-            # tell QT to interpret C strings as utf-8
-            utf8 = QtCore.QTextCodec.codecForName("utf-8")
-            QtCore.QTextCodec.setCodecForCStrings(utf8)
+            app = QtGui.QApplication.instance()
+            if app is None:
 
-            # set the stylesheet
-            self._initialize_dark_look_and_feel()
```

Comment:
Don't we still need to initialize the dark look and feel if we're creating the QApp?